src/main/java/christmas/domain/menu/Menu.java

```diff
@@ -0,0 +1,48 @@
+package christmas.domain.menu;
+
+import christmas.domain.Money;
+import java.util.Arrays;
+
+public enum Menu {
+    MUSHROOM_CREAM_SOUP("양송이수프", FoodType.APPETIZER, new Money(6000)),
+    TAPAS("타파스", FoodType.APPETIZER, new Money(5500)),
+    CAESAR_SALAD("시저샐러드", FoodType.APPETIZER, new Money(8000)),
+    T_BORN_STAKE("티본스테이크", FoodType.MAIN_COURSE, new Money(55000)),
```

Comment:
정-말 사소한 부분이지만, 이와 같이 같은 `enum`에서 다시 세부 카테고리로 구분이 되는 경우에는 개행을 넣어 구분을 해 주는 것도 가독성을 위한 좋은 습관이라 생각해요!
```suggestion
    CAESAR_SALAD("시저샐러드", FoodType.APPETIZER, new Money(8000)),
    
    T_BORN_STAKE("티본스테이크", FoodType.MAIN_COURSE, new Money(55000)),
```